### PR TITLE
Additional features for blogs

### PR DESCRIPTION
**Pull request by SaranjeetKaur**
I have added a couple of lines about using admonitions. We might want to add some more features too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +45,27 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+
+#### Admonitions
+
+Admonitions or call-outs are useful for including side content without interrupting the flow of your blog.
+
+- To include an admonition block, start with `!!!` followed by a single [type qualifier](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) keyword.
+- Add the block content on the next line, indented by four spaces.
+- The type qualifier (in titlecase) is the default title of the block. You can add a customised title after the type qualifier in quoted string.
+
+Refer the [admonitions documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to know more about the different types of admonitions and possible customisations.
```
Comment (dc2917) [inline README.md:61]:
```suggestion
Refer to the [admonitions documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to know more about the different types of admonitions and possible customisations.
```

File: README.md [hunk 2]
```diff
@@ -45,3 +45,27 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+
+#### Code blocks
+
+- To add code blocks, begin and end the block with lines containing three backticks.
+- To add syntax highlighting to those blocks, add the language shortcode directly after the opening block.
+- To add a custom title to the code block use `title="<custom title>"` option directly after the language shortcode.
+
+Refer the [code blocks documentation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/) for more customisations.
```
Comment (dc2917) [inline README.md:69]:
```suggestion
Refer to the [code blocks documentation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/) for more customisations.
```

File: README.md [hunk 3]
```diff
@@ -45,3 +45,27 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+- To include an admonition block, start with `!!!` followed by a single [type qualifier](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) keyword.
+- Add the block content on the next line, indented by four spaces.
+- The type qualifier (in titlecase) is the default title of the block. You can add a customised title after the type qualifier in quoted string.
+
+Refer the [admonitions documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to know more about the different types of admonitions and possible customisations.
+
+#### Code blocks
+
+- To add code blocks, begin and end the block with lines containing three backticks.
+- To add syntax highlighting to those blocks, add the language shortcode directly after the opening block.
```
Comment (dc2917) [inline README.md:66]: Probably worth providing a link to a list of supported languages and their shortcode

File: README.md [hunk 4]
```diff
@@ -45,3 +45,27 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+- To add a custom title to the code block use `title="<custom title>"` option directly after the language shortcode.
+
```
Comment (dc2917) [inline README.md:68]: Might be worth adding a short example to show how to format all of this, e.g. to show that the language name needs to be on the same line as the opening backticks etc

**Review by dc2917 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -45,3 +45,27 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+Refer the [code blocks documentation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/) for more customisations.
+
+#### Customise images
```
Comment (AdrianDAlessandro) [inline README.md:71]: For customising Images, I think links to the following is enough:
- [MkDocs Caption](https://tobiasah.github.io/mkdocs-caption/)
- [Attribute Lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) in the Material Theme docs

File: README.md [hunk 6]
```diff
@@ -45,3 +45,44 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+```` markdown
+``` py title="add_numbers.py"
+def add(x, y, z):
+  return x + y + z
+```
+````
+
+Refer to the [code blocks documentation](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/) for more customisations.
+
+#### Customise images
```
Comment (dc2917) [inline README.md:86]:
```suggestion
#### Customising images
```

File: README.md [hunk 7]
```diff
@@ -45,3 +45,44 @@ Note: If authors are not already listed in `docs/.authors.yml`, please add them.
...
+    massa, nec semper lorem quam in massa.
+````
+
+Refer to the [admonitions documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to learn more about the different types of admonitions and possible customisations.
+
+#### Code blocks
+
+- To add code blocks, begin and end the block with lines containing three backticks.
+- To add syntax highlighting to those blocks, add the language shortcode directly after the opening block. Refer to the [list of available lexers](https://pygments.org/docs/lexers/) to find the shortcode for a given language.
+- To add a custom title to the code block use `title="<custom title>"` option directly after the language shortcode. Here is an example of a Python code block that adds three numbers:
```
Comment (dc2917) [inline README.md:75]:
```suggestion
- To add a custom title to the code block use `title="<custom title>"` option directly after the language shortcode.

Here is an example of a Python code block that adds three numbers:

```

**Review by dc2917 — COMMENTED** [review]
Just a couple of tiny changes but otherwise looks good to me!

**Review by dc2917 — APPROVED** [review]

**Comment by SaranjeetKaur** [comment]
Thanks folks! I will merge this PR now.

**Pull request merged**